script.js

```diff
@@ -0,0 +1,197 @@
+let todoList = [];
+let doneList = [];
+
+// 입력된 값을 todo list에 추가
+const addTodoItem = () => {
+  const todoContent = document.querySelector(".todo-input").value;
+  if (todoContent) paintTodoItem(todoContent);
+};
+
+// 화면에 todo item을 추가
+const paintTodoItem = (text) => {
+  const todoId = new Date().valueOf();
```

Comment:
getTime도 ✨유니크✨한 아이디를 얻기 위해 사용가능합니다. 둘이 동일하다고 하더라구요
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTime

```suggestion
  const todoId = new Date().getTime();
```

Reply:
링크까지 공유해주시다니.. 알려주셔서 감사합니다😄